wandb/sdk/wandb_run.py

```diff
@@ -3464,44 +3443,46 @@ def log_model(
         name: str | None = None,
         aliases: list[str] | None = None,
     ) -> None:
-        """Logs a model artifact containing the contents inside the 'path' to a run and marks it as an output to this run.
+        """Logs a model artifact containing the contents inside the `path` to a run and marks it as an output to this run.
```

Comment:
Wordsmithing for conciseness:

> Logs a model artifact as an output of this run.

The details about "path" are in its Args section.